components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java

```diff
@@ -218,32 +218,31 @@ public String getBusinessOwnerName(String appId) throws AppManagementException {
     /**
      * Delete a given business owner.
      *
-     * @param ownerId
-     * @return
+     * @param businessOwnerId
      */
-    public void deleteBusinessOwner(String ownerId) throws AppManagementException {
+    public void deleteBusinessOwner(String businessOwnerId) throws AppManagementException {
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/4642888b8ad6934dcda162142607b8549b0728e5